src/components/navigation/TopicPopup.js

```diff
@@ -0,0 +1,79 @@
+import _ from 'lodash'
+import React, { Component } from 'react'
+import styles from './TopicPopup.scss'
+import classNames from 'classnames'
+import { LINK_PREFIX, CHARACTERS_LIMIT, CONTINUE_READING } from '../../constants/index'
+import { shortenString } from '../../lib/string-processor'
+import closeIcon from '../../../static/asset/icon-navbar-close.svg'
+
+const Topic = (props) => {
+  const { data } = props
+  const link =  LINK_PREFIX.ARTICLE + data.slug
+  const heroImgUrl = _.get(data, [ 'heroImage', 'image', 'resizedTargets', 'mobile', 'url' ], null)
+
+  return (
+    <div className="col-md-12 col-lg-6">
+      <div className={styles['topic']}>
+        <div className={styles['img-outer']}>
+          <div className={styles['img-box']}>
+            <img className={styles['crop']} src={heroImgUrl}/>
+          </div>
+        </div>
+        <div className={styles['text-box']}>
+          <a href={link}><h3 className={styles['article-title']}>{data.title}</h3></a>
+          <p className={styles['article-desc']}>{shortenString(data.ogDescription, CHARACTERS_LIMIT.TOPIC_DESC)}</p>
```

Comment:
Are we going to shorten the description to 120 words no matter what device user has?